docs/decisions/0006-keep-eligibility-screener-and-recert-portal-separate.md

```diff
@@ -0,0 +1,41 @@
+# Keep the Eligibility Screener and the Participant Recertification Portal separate
+
+* Status: proposed
+* Deciders: @rocketnova, @microwavenby, ...
+* Date: [YYYY-MM-DD when the decision was last updated] <!-- optional -->
+
+Technical Story: https://wicmtdp.atlassian.net/browse/PRP-52
+
+## Context and Problem Statement
+
+Now that the [Eligibility Screener](https://github.com/navapbc/wic-mt-demo-project-eligibility-screener) is being re-written in Remix, the framework that we chose in [ADR-0005](0005-use-remix-for-project-architecture.md) to use for the Participant Recertification Portal,
+
+## Decision Drivers
+
+* Is there an easy way to keep the eligibility screener's dependencies up to date?
+* Is there an easy way to port improvements made in PRP to the eligibility screener?
+* Will it be easier or harder for WIC agencies to use or adopt these tools if they are combined into the same codebase?
+
+## Considered Options
+
+* Combine the eligibility screener and the recertification portal into the same codebase
+* Keep the eligibility screener and the recertification portal as two separate codebases
+
+## Decision Outcome
+
+...
+
+## Pros and Cons of the Options
+
+### Combine the eligibility screener and the recertification portal into the same codebase
+
+* Good, because it will be easy to keep the eligibility screener's dependencies up to date
+* Good, because we can more easily create and manage a library of shared React components
+* Good, because we have fewer code repos to manage
```

Comment:
* Bad, because it may be hard for an interested party to try out one tool without the code and documentation for the other
* Bad, because we might need to build a setup script / wizard to help someone navigate deployment options

Reply:
Bad, because pull requests that impact one tool may impact the other